python/python/lancedb/query.py

```diff
@@ -135,6 +135,8 @@ def create(
         vector_column_name: str,
         ordering_field_name: str = None,
         fts_columns: Union[str, List[str]] = None,
+        *,
+        use_tantivy=True,
```

Comment:
This is query, we should be able to infer what fts index it has?

For example, if we have native index, whether set this `use_tantive = True | False` does not matter, IIRC?

Reply:
yes, this param isn't exposed to users, would set it by existence of tantivy index (true if exists)
see t`able.py`'s `search()` method

Reply:
lets put the this logic right before you call `scan ` or `tantivy`, to reduce the context between the functions.

Reply:
Removed